pickee_frontend/src/components/RecommendationBox/RecommendationBox.vue

```diff
@@ -1,5 +1,6 @@
 <template>
     <div class="recommendation-box">
+        <v-btn color="secondary" class="recommendation-box__goback" href="/?step=2">Back to the preferences</v-btn>
```

Comment:
I would remove 'the' from the button text, so it's 'Back to preferences'. This is obviously a very minor Pickee comment though ;-)

Reply:
Thanks, my english degrades with every PR on this project. 😄 
Done